src/backend/storeManagers/legendary/games.ts

```diff
@@ -637,12 +637,15 @@ export async function install(
   const anticheatInfo = gameAnticheatInfo(getGameInfo(appName).namespace)
 
   if (anticheatInfo && isLinux) {
-    const gameSettings = await getSettings(appName)
+    const gameConfig = GameConfig.get(appName)
+
+    if (anticheatInfo.anticheats.includes('Easy Anti-Cheat')) {
+      gameConfig.setSetting('eacRuntime', true)
+      if (isFlatpak) gameConfig.setSetting('useGameMode', true)
```

Comment:
We had issues in the past with this approach, is safer to just use brackets like in other places.